app/processors/gitlab_processor.py

```diff
@@ -45,8 +56,30 @@ def urlencode_paths(path):
     return path
 
 
+def fix_link_header(headers):
+    """Replace the GitLab URL in the Link header."""
+
+    if 'Link' in headers:
+        headers['Link'] = headers['Link'].replace(
+            app.config['GITLAB_URL'], app.config['HOST_NAME']
+        )
+
+    return headers
+
+
 class GitlabGeneric(BaseProcessor):
 
+    def __init__(self, path, endpoint):
+        self.path = path
```

Comment:
why not `super().__init__(path, endpoint)`

Reply:
yep, you're right!